network/ipam/reference-content/ipv6.mdx

```diff
@@ -0,0 +1,129 @@
+
+As well as providing a much bigger address space, IPv6 also includes a built-in network security layer (IPsec), as well as improved features for reliability and efficiency, liked autoconfiguration, streamlined headers and improved Quality of Service (QoS). All leading to a more robust and secure protocol, that can potentially offer lower latency and faster data transfer. 
+
+## IPv6-compatible products
+
+The following products support IPv6:
+
+### Instances and IPv6
+
+Scaleway Instances are compatible with IPv6, with the caveat that the Instance must be using [routed IPs](/compute/instances/concepts/#routed-flexible-ip). Recently created Instances use routed IPs by default, older Instances may need to be [moved manually to routed IPs](/compute/instances/how-to/migrate-routed-ips/).
+
+#### Public IPv6 
+
+You can attach one or multiple public (flexible) IPv6 addresses to your Instance, as well as, or instead of, a public IPv4 address. These public addresses are flexible, meaning that you can detach them from an Instance, hold them in your account, and attach them to a different Instance later, if you want. Each flexible IPv6 address is a `/64` IPv6 subnet.
+
+#### Private IPv6 
+
+When you attach an Instance to a Private Network, it gets both an IPv4 and an IPv6 address on that network. You can either let IPAM auto-allocate any available address, or specify a [reserved IP address](/network/ipam/how-to/reserve-ip/) to use.
+
+#### Going further
+
+Go further with Instances and IPv6 in the following documentation:
+
+- [How to use flexible IPs](/compute/instances/how-to/use-flexips/)
+- [Compatibility between OS images and different flexible IP type combinations](/compute/instances/reference-content/comaptibility-scw-os-images-flexible-ip/)
+- [Using routed IPs](/compute/instances/api-cli/using-routed-ips/)
+- Fix lost IPv6 connectivity when migrating to routed IP for old [Debian Buster images](/instances/troubleshooting/fix-lost-ip-connectivity-on-debian-buster/) or [RHEL images](/compute/instances/troubleshooting/fix-unreachable-ipv6-rhel-based-instance/)
+- [Fix DNS resolution with a routed IPv6-only setup on Debian Bullseye](/compute/instances/troubleshooting/fix-dns-routed-ipv6-only-debian-bullseye/)
+
+### Elastic Metal and IPv6
+
+Scaleway Elastic Metal servers are compatible with IPv6. 
+
+#### Public IPv6
+
+You can attach one or multiple public (flexible) IPv6 addresses to your Elastic Metal server, as well as, or instead of, a public IPv4 address. These public addresses are flexible, meaning that you can detach them from an Elastic Metal server, hold them in your account, and attach them to a different Elastic Metal server, if you want. Each flexible IPv6 address is a `/64` IPv6 subnet. Flexible IPv6 addresses can also be used as additional IP addresses to create virtual machines on your Elastic Metal server.
```

Comment:
Each Elastic Metal server comes with one fixed IPv4 that is not detachable (possible to deactivate in software, but the IP remains assigned to the machine)

Reply:
if I change it to " as well as, or instead of, a flexible IPv4 address" would that work?

Or alternatively I could add a note message box that says "Each Elastic Metal server comes with a fixed (not flexible) IPv4 that is not detachable, though this can be deactivated manually".